templates/settings.py.erb

```diff
@@ -19,9 +19,12 @@ DATABASES = {
 <% end -%>
     },
 }
+
+<% if scope['pulpcore::use_rq_tasking_system'] || scope['pulpcore::cache_enabled'] -%>
 REDIS_HOST = "localhost"
 REDIS_PORT = "<%= scope['redis::port'] %>"
 REDIS_DB = <%= scope['pulpcore::redis_db'] %>
+<% end -%>
 
```

Comment:
I'd have the empty line within the block. That way `settings.py` can't end up with 2 blank lines:
```suggestion

<% end -%>
```